hadoop-hdds/interface-client/src/main/proto/hdds.proto

```diff
@@ -514,6 +514,14 @@ message DeletedBlocksTransactionInfo {
     optional int32 count = 4;
 }
 
+message DeletedBlocksTransactionSummary {
+    optional int64 firstTxID = 1; // starting ID of transaction to be counted into summary
```

Comment:
Is this useful after the upgrade and old transaction is removed? If not then may not be requried, count itself can tell last n transaction used for summation.